lib/src/over_react_test/common_component_util.dart

```diff
@@ -249,15 +263,59 @@ void _testPropForwarding(BuilderOnlyUiFactory factory, dynamic childrenFactory()
     for (var forwardingTarget in forwardingTargets) {
       Map actualProps = getProps(forwardingTarget);
 
-      // If the forwarding target is a DOM element it will should not have invalid DOM props forwarded to it.
+      // If the forwarding target is a DOM element it should not have invalid DOM props forwarded to it.
       if (isDomElement(forwardingTarget)) {
         otherProps.forEach((key, value) {
-          expect(actualProps, isNot(containsPair(key, value)));
+          expect(actualProps[key], isNull, reason: unindent('''
+            The following mock key/value pair(s) added by this test were found on 
+            a DOM component that props were forwarded to:
+            
+                $key: ${actualProps[key]}
+                 
+            This means that `addUnconsumedProps` is mistakenly being used 
+            instead of `addUnconsumedDomProps` on a DOM component.
+          '''));
         });
       } else {
         otherProps.forEach((key, value) {
           expect(actualProps, containsPair(key, value));
         });
+
+        final missingUnconsumedPropKeys = unconsumedProps.keys.where((key) => !actualProps.containsKey(key));
+        final mixinNamesOfMissingUnconsumedPropKeys = missingUnconsumedPropKeys.map(getPropKeyNamespaceFromPropKey).where((name) => name != null).toSet();
```

Comment:
This excludes un-namespaced keys, or keys with different namespaces than their parent classes, which could result in confusing error messages.

I think we should get rid of `getPropKeyNamespaceFromPropKey` and just say `the mixin with key "FooProps.foo"` instead of trying to infer that it's from `FooProps`.

Reply:
Addressed in https://github.com/Workiva/over_react_test/pull/102/commits/7a98196fc81981ba1d0c586c0d6d896709934108